### ISSUE TITLE
Episcanpy 0.1.8 should require pysam

### ISSUE THREAD

**Issue by le-ander** [colomemaria/epiScanpy#52]
Hi Anna,

when importing episcanpy 0.1.8, I get the following error:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/app/python37/lib/python3.7/site-packages/episcanpy/__init__.py", line 25, in <module>
    from . import count_matrix as ct
  File "/app/python37/lib/python3.7/site-packages/episcanpy/count_matrix/__init__.py", line 6, in <module>
    from ..count_matrix._bld_atac_mtx import bld_mtx_fly
  File "/app/python37/lib/python3.7/site-packages/episcanpy/count_matrix/_bld_atac_mtx.py", line 5, in <module>
    import pysam
ModuleNotFoundError: No module named 'pysam'
```

Once I manually install pysam, this error disappears.
I guess pysam should be added to the requirements file of episcanpy.

**Comment by kridsadakorn** [comment]
If you want to install the version on Github, this configuration may help

RUN conda install -y \
        conda==4.7.10 \
    && conda clean -ya
RUN conda install -y --channel=bioconda --channel=conda-forge \
        jupyter==1.0.0 \
        louvain==0.6.1 \
        numba==0.45.0 \
        python-igraph==0.7.1.post7 \
        scipy==1.4.1 \
        seaborn==0.9.0 \
        statsmodels==0.10.0 \
    && conda clean -ya

RUN pip install -U --no-cache-dir --upgrade-strategy=only-if-needed \
    cmake==3.13.3 \
    matplotlib==3.1.0 \
    multicoretsne==0.1 \
    numpy==1.16.4 \
    pandas==0.24.2 \
    pysam==0.15.3 \
    scanpy==1.4.4.post1 \
    scikit-learn==0.21.2 \
    -e git+https://github.com/colomemaria/epiScanpy#egg=2bd1a9e235693251adc60d73ac5fe6abf76729fc

**Comment by le-ander** [comment]
this has been fixed in 4bf21a3eea1bce43516c55d4f8f9677e6816b3d8
